lib/src/kvs/tx.rs

```diff
@@ -2722,7 +2740,7 @@ impl Transaction {
 	// complete_changes will complete the changefeed recording for the given namespace and database.
 	//
 	// Under the hood, this function calls the transaction's `set_versionstamped_key` for each change.
-	// Every change must be recorded by calling this struct's `record_change` function beforehand.
```

Comment:
Just asking- Is a struct with methods called `model` in Rust?! (It reminds me of Rails but I'm not sure if that's the universal term.

Reply:
Model I think comes from MVC mostly. It is what I am used to from java and go 😂

Reply:
Ah sorry! This was a rename from the editor, I am reverting!